### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by k3ntako**
1) `issueSnackbar` wasn't being passed down
2) changed API end point to `/inbound/teams` instead of `/inbound/team` to be more consistent with the email address `@teams.wakebird.com`
3) Removed custom email CSS, which was used for matching styles in Draft.js

### CODE REVIEW

File: client/pages/Team/EmailTab.js [hunk 1]
```diff
@@ -176,9 +179,7 @@ class EmailTab extends Component {
 
 
 export default function(props){
-  return <SnackbarConsumer>
-    { issueSnackbar => <MeConsumer>
-      { meState => <EmailTab {...props} issueSnackbar={issueSnackbar} me={meState.me} /> }
-    </MeConsumer> }
-  </SnackbarConsumer>
+  return <MeConsumer>
```
Comment (mienaikoe) [inline client/pages/Team/EmailTab.js]: What's the point of moving a Consumer from the default export to the render?

File: server/routes.js [hunk 2]
```diff
@@ -174,7 +174,7 @@ router.route( '/emails/:email_id/exclude').
   post(                                                   EmailController.exclude);
 
 // Inbound Emails
-router.route( '/inbound/team').
+router.route( '/inbound/teams').
```
Comment (mienaikoe) [inline server/routes.js]: Just to be sure, you changed this URL in sendgrid as well, right?

File: client/pages/Team/EmailTab.js (hunk 1 above)
Comment (k3ntako) [reply client/pages/Team/EmailTab.js]: It's not being used anywhere else, so I thought it was better to minimize it's scope

Comment (mienaikoe) [reply client/pages/Team/EmailTab.js]: That doesn't save any memory or processing time, and it makes the code harder to read.

Additionally, if something else in this file needed to use the consumer you would have to refactor the class.

Comment (k3ntako) [reply client/pages/Team/EmailTab.js]: I can move it out

**Pull request merged**